modules/infotext_utils.py

```diff
@@ -230,6 +231,47 @@ def restore_old_hires_fix_params(res):
     res['Hires resize-2'] = height
 
 
+def parse_civitai_resources(x: str):

```

Comment:
I think this should be implemented in an extension using `script_callbacks.on_infotext_pasted`